src/Microsoft.FeatureManagement/Targeting/AudienceExclusion.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System.Collections.Generic;
+
+namespace Microsoft.FeatureManagement.FeatureFilters
+{
+	/// <summary>
+	/// Excludes users and groups from the audience.
```

Comment:
The type itself should really just define users and groups. The exclusion part should come from it's presence as the `Exclude` property of `Audience.`

Reply:
Treating this comment as a duplicate of https://github.com/microsoft/FeatureManagement-Dotnet/pull/218#discussion_r1145479412